neurom/features/neuritefunc.py

```diff
@@ -308,14 +326,50 @@ def partition_asymmetries(neurites, neurite_type=None):
 
 def partition_pairs(neurites, neurite_type=None):
     '''Partition pairs at bifurcation points of a collection of neurites.
-    Partition pait is defined as the number of bifurcations at the two
+    Partition pair is defined as the number of bifurcations at the two
     daughters of the bifurcating section'''
     return map(bifurcationfunc.partition_pair,
                iter_sections(neurites,
                              iterator_type=Tree.ibifurcation_point,
                              neurite_filter=is_type(neurite_type)))
 
 
+def partition_asymmetry_lengths(neurites, neurite_type=None):
```

Comment:
Merge it with `partition_asymmetry` and add a variant argument.
Then in `__init__.py`:
```
from functools import partial
'partition_asymmetry_length': partial(_nrt.partition_asymmetry, variant='length')
```

Reply:
Just to make sure there is no mismatch between the `tot_lengths` array and the iter_neurite(...) iterator, I would call `total_length_per_neurite` individually for each neurite. Actually there was a potential mismatch in your case as you have a neurite_type filter in `iter_neurites` but not in `total_length_per_neurite`.
```python
asymmetries = list()
for neurite in iter_neurites(neurites, filt=is_type(neurite_type)):
     neurite_length = total_length_per_neurite(neurite)
     for section in iter_sections(neurite,
                                        iterator_type=Tree.ibifurcation_point,
                                        neurite_filter=is_type(neurite_type))):
         pathlength_diff = downstream_pathlength(section.child[0]) - downstream_pathlength(section.child[1])
         asymetries.append(abs(pathlength_diff) / neurite_length)
```